sparse-11/usr/bin/droid/droid-hal-early-init.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if ! grep -q hybris /system/etc/ld.config.29.txt; then
+    mount -o bind /usr/libexec/droid-hybris/system/etc/ld.config.29.txt /system/etc/ld.config.29.txt
```

Comment:
Isn't it API level 30?

Reply:
This method doesn't work anymore, I will move the file.